### PR TITLE
docs(extensions): Prefix titles with extension type term

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/100-model.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Add custom methods to your models'
+title: '`model`: Add custom methods to your models'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/100-model.mdx:2]:
```suggestion
title: '<inlinecode>model</inlinecode>: Add custom methods to your models'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/110-client.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Add methods to Prisma Client'
+title: '`client`: Add methods to Prisma Client'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/110-client.mdx:2]:
```suggestion
title: '<inlinecode>client</inlinecode>: Add methods to Prisma Client'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Create custom Prisma Client queries'
+title: '`query`: Create custom Prisma Client queries'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx:2]:
```suggestion
title: '<inlinecode>query</inlinecode>: Create custom Prisma Client queries'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Add custom fields and methods to query results'
+title: '`result`: Add custom fields and methods to query results'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx:2]:
```suggestion
title: '<inlinecode>result</inlinecode>: Add custom fields and methods to query results'
```

**Comment by ruheni** [comment]
It seems the `<inlinecode>` component can't be used in the front matter. This is what it looks like atm:

CleanShot 2023-09-25 at 11 23 53@2x

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/100-model.mdx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Add custom methods to your models'
+title: '<inlinecode>model</inlinecode>: Add custom methods to your models'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/100-model.mdx]:
```suggestion
title: '`model`: Add custom methods to your models'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/110-client.mdx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Add methods to Prisma Client'
+title: '<inlinecode>client</inlinecode>: Add methods to Prisma Client'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/110-client.mdx]:
```suggestion
title: '`client`: Add methods to Prisma Client'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Create custom Prisma Client queries'
+title: '<inlinecode>query</inlinecode>: Create custom Prisma Client queries'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx]:
```suggestion
title: '`query`: Create custom Prisma Client queries'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Add custom fields and methods to query results'
+title: '<inlinecode>result</inlinecode>: Add custom fields and methods to query results'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx]:
```suggestion
title: '`result`: Add custom fields and methods to query results'
```

**Review by ruheni — COMMENTED** [review]
While the change makes it easier to find information, I'm not a fan of it.

What is the main motivation for the change? Are the page titles not specific enough?

CleanShot 2023-11-16 at 11 41 15@2x

**Comment by janpio** [comment]
Yes, the current titles do not clearly express what extension type they are talking about. We are missing the clear: model, client, query, result.

Suggestions welcome on how to do that in other ways, otherwise I suggest approving and merging.

**Pull request merged**